compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "luci/CircleOptimizer.h"
 
+#include "luci/Pass/FoldDequantize.h"
```

Comment:
(optional) It would be better to follow the naming convention. :) `FoldDequantizePass.h`

Reply:
Oops It's totally my mistake, thanks!